application/web/index.html

```diff
@@ -42,7 +42,18 @@
                         <label class="new_entry">Create New Entry</label>
                     </div>
                 </button>
-
+                <div class = "home-list">
+                    <div class="home-entry-descriptor">
+                        <span>Entry Name</span>
+                        <span>Date Logged</span>
+                    </div>
+                    <button class="home-single-entry">
+                        <span class="home-entry-name">My Journal</span>
+                        <span>Date Created</span>
+                        <img src="/application/web/assets/vdots-entry-black.png">
```

Comment:
Is it just me or the 3 dots png isn't displaying?
![image](https://github.com/cse110-sp24-group3/cse110-sp24-group3/assets/101247244/f916ee62-9d32-4f5f-9b7b-bf1d0806de3d)